neo/SmartContract/Native/Votes/Interface/IMultiVoteModel.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Neo.SmartContract.Native.Votes.Interface
+{
+    interface IMultiVoteModel
+    {
+        //TODO: details for interface
+        int[,] CalculateVote(List<CalculatedMultiVote> votes);
+    }
+
+    public class CalculatedMultiVote
+    {
+        public int balance;
+        public List<int> vote;
+        public CalculatedMultiVote(int Balance, List<int> Vote)
```

Comment:
I think it could be _balance and _vote as arguments, but I am not sure about what is the standard. @shargon

Reply:
`list<int> votes`